pages/create.tsx

```diff
@@ -83,13 +83,16 @@ export default function Create() {
     let zeroFee = false;
     for (let i = 0; i < checkedState?.length; i++) {
       const currUtxo = utxoList[i];
-      if (
+      // cannot remix with multiple utxos
+      if (zeroFee && checkedState[i]) {
+        zeroFee = false;
+        break;
```

Comment:
If multiple UTXOs are selected and the UTXO with `anonSet > 1` is last in the list, `zeroFee` will evaluate to false because it won't get here